### DISCUSSION TITLE
Mdx running into an error

### DISCUSSION THREAD

**Discussion by gayathrirajendran** [storybookjs/storybook#17033] [Help]
**package.json** 
"@storybook/addon-docs": "^6.3.10",
"@storybook/addon-essentials": "^6.3.10",
"@storybook/addon-links": "^6.3.10",
"@storybook/angular": "^6.3.10",
"@storybook/builder-webpack5": "^6.3.10",
"@storybook/manager-webpack5": "^6.3.10",

**main.js**
module.exports = {
  "stories": [
    "../src/**/*.stories.mdx",
    "../src/**/*.stories.@(js|jsx|ts|tsx)",
    "src/stories/**/*.stories.@(ts|tsx)",
    "../src/stories/**/*.stories/index.@(ts|tsx)",
    "projects/cbxds-ui/**/*.stories.@(ts|tsx)",
    "projects/**/*.stories.@(js|jsx|ts|tsx)",
  ],
  "addons": [
    "@storybook/addon-links",
    "@storybook/addon-essentials",
    '@storybook/addon-a11y',
    '@etchteam/storybook-addon-status',
    '@storybook/addon-actions',
    "@storybook/addon-design-assets",
    "@storybook/addon-console",
    {
      name: '@storybook/addon-docs',
      options: {
        configureJSX: true,
        babelOptions: {},
        sourceLoaderOptions: null,
        transcludeMarkdown: true,
      },
    }
  ],
  "core": {
    "builder": "webpack5"
  }
}

Am trying to create an mdx file internationalization.mdx
import { Meta, Description } from '@storybook/addon-docs';
import readme from 'Internationalization.md'

<Meta title="Example/Introduction" />

<Description markdown={readme} />

Am getting an error like below
No `docs.container` set, did you run `addon-docs/preset`? ; Zone: <root> ; Task: Promise.then ; Value: Error: No `docs.container` set, did you run `addon-docs/preset`

Can you help ?

Even normal mdx with basic markdown no embeds is not working, fails with the same message

**Comment by shilman** [comment]
Do you a have a reproduction repo you can share? If not, can you create one? See [how to create a repro](https://storybook.js.org/docs/react/contribute/how-to-reproduce).

**Reply by gayathrirajendran** [reply]
When trying to setup a reproduction, the scripts are forcing yarn update to berry and messes with global settings that are on yarn classic that I need.
Will share one when I find time, meanwhile a normal .mdx with just markdown content and Meta tag works, embed of an md file in mdx not working, it throws no errors either. The error that I talked about in the discussion, went away when I removed addon-docs from dependencies maybe because I already have addon-essentials.

**Reply by blowsie** [reply]
I can reproduce this issue here
https://github.com/blowsie/vue3-vitesse-storybook-pnpm/tree/master/packages/ui
